Frends.ServiceBus.Read/Frends.ServiceBus.Read/Frends.ServiceBus.Read.csproj

```diff
@@ -12,7 +12,7 @@
 	<GenerateDocumentationFile>true</GenerateDocumentationFile>
```

Comment:
https://github.com/FrendsPlatform/Frends.CrossplatformServiceBus/blob/a153142f6a5cefde148de4827cc8408c49e6a3ef/Frends.ServiceBus.Read/Frends.ServiceBus.Read/Frends.ServiceBus.Read.csproj#L5
Task is not released yet, so the version can be lowered back to 1.0.0.

https://github.com/FrendsPlatform/Frends.CrossplatformServiceBus/blob/a153142f6a5cefde148de4827cc8408c49e6a3ef/Frends.ServiceBus.Read/CHANGELOG.md?plain=1#L3
This entry can also be removed, since all of this is still initial implementation, since the task is not yet released.